dev/customer_testing/ci.bat

```diff
@@ -19,7 +19,7 @@ ECHO.
 ECHO Finding correct version of customer tests...
 CMD /S /C "IF EXIST "..\..\bin\cache\pkg\tests\" RMDIR /S /Q ..\..\bin\cache\pkg\tests"
 git clone https://github.com/flutter/tests.git ..\..\bin\cache\pkg\tests
-FOR /F "usebackq tokens=*" %%a IN (`dart --enable-asserts ..\tools\bin\find_commit.dart ..\..\bin\cache\pkg\tests`) DO git -C ..\..\bin\cache\pkg\tests checkout %%a
+FOR /F "usebackq tokens=*" %%a IN (`dart --enable-asserts ..\tools\bin\find_commit.dart . master ..\..\bin\cache\pkg\tests main`) DO git -C ..\..\bin\cache\pkg\tests checkout %%a
```

Comment:
can you explain this diff? i have no batch foo.

Reply:
oh wait, nvm, i see the diff on find_commit.dart

Reply:
Is sending two more parameters to find_commit.

`find_commit <first repo checkout in this case flutter/flutter> <branch of first repo (master)> <second repo checkout in this case flutter/tests> <branch of second repo(main)>`

Reply:
ahh i see, confusingly in the old version, the first arg was optional. this LGTM